resources/apigatewayv2-apis.go

```diff
@@ -0,0 +1,60 @@
+package resources
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/apigatewayv2"
+)
+
+type APIGatewayV2API struct {
+	svc     *apigatewayv2.ApiGatewayV2
+	v2APIID *string
+}
+
+func init() {
+	register("APIGatewayV2API", ListAPIGatewayV2APIs)
+}
+
+func ListAPIGatewayV2APIs(sess *session.Session) ([]Resource, error) {
+	svc := apigatewayv2.New(sess)
+	resources := []Resource{}
+
+	params := &apigatewayv2.GetApisInput{
+		MaxResults: aws.String("100"),
```

Comment:
I have absolutely no idea why one would use a string for such an option
https://docs.aws.amazon.com/sdk-for-go/api/service/apigatewayv2/#GetApisInput

Reply:
My guess is that something with API generation went wrong and that cannot get reverted without breaking other peoples stuff.